converter/isosts2nisosts/README.md

```diff
@@ -0,0 +1,4 @@
+#ISOSTS to NISO-STS Converter
```

Comment:
Please add a description what it does, especially that it is only modifying the metadata section.